styleguide/derek/examples/landing-page.ejs

```diff
@@ -33,3 +33,9 @@
         </div>
     </div>
 </div>
+<script src="https://577351e10bc4a0198d93-f60a0bb748a3c84145bb10da7563bafb.ssl.cf1.rackcdn.com/test/rackspace-js-lib-ticker.js?mo"></script>
```

Comment:
Is this `test` folder on the cdn going to be the final location of this javascript file, or is this the URL someone else gave to you? I'm just curious.